src/js/stores/MarathonStore.js

```diff
@@ -56,7 +63,8 @@ var MarathonStore = Store.createStore({
   mixins: [GetSetMixin],
 
   getSet_data: {
-    apps: {}
+    apps: {},
+    deployments: []
```

Comment:
https://github.com/dcos/dcos-ui/pull/140/files#diff-8b6eb6158535aefd0b151ff7df2802a3R67

(in PR #140)